.vscode/tasks.json

```diff
@@ -5,8 +5,8 @@
   "tasks": [
     {
       "taskName": "Watch and compile TypeScript",
-      "command": "tsc",
-      "args": ["--watch"],
+      "command": "npm",
+      "args": ["--silent", "run", "build:watch"],
```

Comment:
I find this change suspicious, I hope it will go away after you rebase this patch on the latest master (that will include #10).

Reply:
it should. will be rebasing shortly.